covid19-notebooks/welcome.html

```diff
@@ -4,63 +4,51 @@
     <meta charset="UTF-8">
     <title>Welcome Page</title>
     <style>
-        .header {
+        body {
             font-family: "Source Sans Pro", sans-serif;
-            padding: 20px;
-            background : white;
             color : black;
+            background: rgb(245, 245, 245);
+            margin: 0;
+        }
+        .header {
+            padding: 20px 20px 0;
+            margin: 0 0 20px;
+            background : #FFF;
             display: flex;
             justify-content: center;
             align-items: center;
         }
         .header h1 {
-            font-family: "Source Sans Pro", sans-serif;
             font-size: 3vw;
             padding-left: 10px;
         }
-
         img {
             width: 50%;
         }
-
-        .bar {
-            width: 100%;
-            height: 30px;
-            background-color: #421C52
-        }
         .content {
-            padding-left: 10px;
-            font-family: "Source Sans Pro", sans-serif;
-        }
-
-        .footer {
-            width: 99%;
-            height: 30px;
-            background-color: #421C52;
-            position: fixed;
-            bottom: 0pt
+            padding: 1px 10px 10px 10px;
+            background : #FFF;
+            margin: 0 8px;
+            border-left-color:rgb(173, 145, 255);
+            box-shadow: 0 2px 4px rgba(0, 0, 0, 0.3);
         }
     </style>
 </head>
 <body>
 <div class="header">
-    <img src = "https://qa-covid19.planx-pla.net/dashboard/Public/sponsors-logo/chicagoland_covid19_commons.png">
-    <h1><b>Welcome to the Chicagoland Pandemic Response Commons</b></h1>
-</div>
-
-<div class="bar">
+    <img src = "https://qa-covid19.planx-pla.net/dashboard/Public/sponsors-logo/chicagoland_covid19_commons.png" alt="Chicagoland COVID-19 Commons"/>
```

Comment:
reference prod instead of qa-covid19

Reply:
thanks forgot to take it out after testing, but now its gone all-together